### ISSUE TITLE
Verify behavior of va-breadcrumbs when used in conjunction with React Router links

### ISSUE THREAD

**Issue by it-harrison** [department-of-veterans-affairs/vets-design-system-documentation#3345]
Recently, [a question](https://github.com/department-of-veterans-affairs/component-library/pull/979/#pullrequestreview-1780483832) was raised about the desired behavior of react router links in the `va-breadcrumb` component. At a high level the question is whether the last rendered breadcrumb should always correspond to the current page or if (as when using react router links) this does not have to be the case.

The goal of this ticket is to determine the correct behavior of the component in this situation and to make any necessary changes.

**Comment by caw310** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Design-System-5f8de67192551b0012ebb802/issues/department-of-veterans-affairs/vets-design-system-documentation/3345?planning-poker) with Zenhub @Andrew565 @ataker @harshil1793 @it-harrison @jamigibbs @micahchiang @powellkerry @rsmithadhoc

**Comment by jamigibbs** [comment]
@humancompanion-usds Yes, it appears to be a duplicate now. While the other ticket was created later, it contains more detail and discussion so I'm going to close this one in favor of that.
